### ISSUE TITLE
All directives are broken - missing a less than sign at the beginning

### ISSUE THREAD

**Issue by abdel-ships-it** [devotis/sublime-angular-material-snippets#11]
Every directive is missing a less than sign at the beginning

Please have a look at this [gif](https://i.gyazo.com/b8565b9ecb7f786e92caf6cf865f4c3f.gif)

**Comment by stephnr** [comment]
Oh how hilariousssssssss. Because we just had this other issue before where there was one too many carets.

**Comment by christiaanwesterbeek** [comment]
Yes. Before this commit https://github.com/devotis/sublime-angular-material-snippets/commit/2d2a476ef9425bef600c64b183df037267683ed7

When in an html file, when I typed md- nothing happens. When I typed <md- I got the snippets menu showing md- at the top, which is fine. When I clicked it, I end up with this snippet. And notice the double caret in the beginning.

```
<<md-tab label="" disabled md-on-select="" md-on-deselect="">...</md-tab>
```

So the commit by @Stephn-R was to fix that, but apprently that makes for @appie18 to have no leading caret at all.

Is there some ST setting that I and @appie18 have different? I have no clue at this point. But I do think that there are tons of sublime snippets libraries out there that must have solved this problem. Is anybody acquainted with a collaborator of one of those sublime snippets libraries that may be willing to help us out?

**Comment by stephnr** [comment]
Here is another popular project using HTML type snippets. Perhaps we might see something there:

https://github.com/joshnh/HTML-Snippets/blob/master/comm.sublime-snippet

**Comment by abdel-ships-it** [comment]
Any updates on this issue?

**Comment by christiaanwesterbeek** [comment]
You're welcome to implement a fix. I don't have time to fixt it myself and what I tried before myself didn't work out for others
